Makefile

```diff
@@ -41,7 +42,11 @@ docker/weave:
 	curl -L git.io/weave -o docker/weave
 	chmod u+x docker/weave
 
-$(SCOPE_EXPORT): $(SCOPE_EXE) $(DOCKER_DISTRIB) docker/weave $(RUNSVINIT) docker/Dockerfile docker/demo.json docker/run-app docker/run-probe docker/entrypoint.sh
+docker/ebpf.tgz:
```

Comment:
I would like that rule to have a dependency so that whenever a developer edit the `Makefile` with a new `EBPF_IMAGE=`, the `docker/ebpf.tgz` makefile target get rebuild.

@krnowak @schu Would it work to just add "Makefile" in the dependency, or do we need to do something else?